test/cypress/integration/Highcharts/mouse-wheel-zoom/zoom-chart.cy.js

```diff
@@ -0,0 +1,41 @@
+describe('Mouse wheel chart zoom, #19178', () => {
+
+    before(() => {
+        cy.visit('/highcharts/cypress/mouse-wheel-zoom');
+    });
+
+    it('Should change yAxis extremes and keep axis options after zoom', () => {
+        cy.chart().then(chart => {
+
+            const yAxis = chart.yAxis[0],
+                { min, max } = yAxis.getExtremes();
+
+            cy.get('#container')
+                .trigger("wheel",
+                    {
+                        deltaY: -66.666666, wheelDelta: 120, wheelDeltaX: 0,
+                        wheelDeltaY: 120, bubbles: true
+                    }
+                ).then(() => {
```

Comment:
I think we should use a different code format, like e.g. here: https://github.com/highcharts/highcharts/blob/bcfc1ad10b53ab5d90bf64b76a8228713d66716e/ts/Core/Tooltip.ts#L1131-L1133
(brackets `{` positions)